garrysmod/gamemodes/terrortown/gamemode/shared.lua

```diff
@@ -218,3 +210,9 @@ DefaultEquipment = {
       "weapon_ttt_glock"
    }
 };
+
+-- Mute types
+MUTE_NONE   = 0
+MUTE_TERROR = TEAM_TERROR
+MUTE_ALL    = 2
+MUTE_SPEC   = TEAM_SPEC

```

Comment:
Actually I just read the code and wow, it actually checks the team against the MUTE enum.
https://github.com/garrynewman/garrysmod/blob/master/garrysmod/gamemodes/terrortown/gamemode/gamemsg.lua#L189
That's kind of silly, why even have this enum? Just use bit flags and check if the Team is present in there.